File: #6 carrinho_de_compras/2018-1/elisson java/Item.java
public class Item {
	private Produto produto;
	private int quantidade;
	
	public Item(Produto produto){
		this.produto = produto;
		this.quantidade = 1;
	}
	
	public float valor() {
		return produto.getPreco() * quantidade;
	}

	public void setQuantidade(int quantidade) {
		this.quantidade = quantidade;
	}
	
	public int getQuantidade() {
		return quantidade;
	}
	
	public Produto getProduto() {
		return produto;
	}
}

File: #6 carrinho_de_compras/2018-1/elisson C#/Item.cs
using System.Collections;
using System.Collections.Generic;

public class Item {
	public Produto Produto { get; set; }
	public int Quantidade { get; set; }
	public Item(Produto produto) {
		this.Produto = produto;
		this.Quantidade = 1;
	}

	public float Total(){
		return this.Produto.Preco * this.Quantidade;
	}
}
